pages/judgement/compile-wrong/index.md

```diff
@@ -0,0 +1,19 @@
+---
+order: 2
+title: 编译错误
+---
+
+可能有以下情况：
+>
+*1. 递交时选错了编程语言
+*2. Java 的主类名没有使用 "Main"
+*3. 对于 C/C++：见下
+*4. 一般性的编译错误
+
+对 C/C++ 选手的特别提醒：
+>
+*1. __int64 在 GNU C++ 中应写成 long long 类型
+*2. main() 返回值必须定义为 int ，而不是 void
+*3. for 语句中的指标变量 i 将会在如"for (int i = *0...) {...}"语句之后变为无效
+*4. itoa 不是一个通用 ANSI 函数（标准 C/C++ 中无此函数）
+*5. printf 中使用 %lf 格式是不正确的
```

Comment:
One more important item:

If you are using `iostream`, you should call `std::ios::sync_with_stdio(false)`. This will greatly improve the I/O performance.